pypesto/ensemble/utils.py

```diff
@@ -230,3 +237,57 @@ def get_prediction_dataset(ens: Union[Ensemble, EnsemblePrediction],
                         'an EnsemblePrediction object as input. Stopping.')
 
     return dataset
+
+
+def read_ensemble_prediction_from_h5(
+        predictor: Union[Callable[[Sequence], PredictionResult], None],
+        input_file: str):
+
+    # open file
+    with h5py.File(input_file, 'r') as f:
+        pred_res_list = []
+        bounds = {}
+        for key in f.keys():
+            if key == 'prediction_id':
+                prediction_id = f[key][()].decode()
+                continue
+            if key in {'lower_bounds', 'upper_bounds'}:
```

Comment:
```suggestion
            if key in {LOWER_BOUND, UPPER_BOUND}:
```
To match a previous suggestion